tests/test_multi_process_lmdb.py

```diff
@@ -0,0 +1,72 @@
+import os
+from multiprocessing import Process
+from pathlib import Path
+from shutil import rmtree
+
+import lmdb
+import pytest
+from lmdb import Error
+from time import sleep
+
+
+class TestDBPath:
+    lmdb = 'test_lmdb'
+
+
+def print_process_info(title):
+    print("\n--------------------------")
+    print(title)
+    print('module name:', __name__)
+    print('parent process:', os.getppid())
+    print('process id:', os.getpid())
+
+
+def f_lmdb(name: str):
```

Comment:
In this test, both processes seems operate as writers. One should work as a Writer and the other as a Reader.